examples/cosmwasm/zero-to-hero/vote.qnt

```diff
@@ -491,7 +491,8 @@ module state {
       val poll = state.polls.get(poll_id)
       poll.options.listForall(option =>
         val optionKey = option._1
-        val optionSum = option._2
+        // FIXME: Type annotation below is a workaround, inferred type is too general
+        val optionSum: int = option._2
```

Comment:
This specific case seems different from the others. I'll look at it in a separate issue (to be created). Every other FIXME of this kind has been removed :smile: